proto/wechaty/puppet.proto

```diff
@@ -13,6 +13,8 @@
 syntax = "proto3";
 package wechaty;
 
+option java_package="io.github.wechaty.puppet";
```

Comment:
Please rename the package name from `io.github.wechaty.puppet` to `io.github.wechaty.grpc`.

Because this is the GRPC module.

The `puppet` module will be different than this repo, and it should be published to another module named `io.github.wechaty.wechaty-puppet`.

Reply:
ok I will fix it